spec.md

```diff
@@ -148,30 +148,39 @@ that contains both context and data).
 
 ### source
 * Type: Object
-* Description: This describes the software instance that emits the event at
-  runtime (i.e. the producer). It contains sub-properties (listed below)
+* Description: This describes the software where an event occurred. It contains sub-properties (listed below)
 * Constraints:
   * REQUIRED
   * MUST contain at least one non-empty sub-property.
 
 ### source-type
 * Type: String
-* Description: Type of the event source. Providers define list of event
-  sources.
+* Description: The kind of object on which an an event occurred. Providers define list of source-types.
 * Constraints:
   * REQUIRED
   * MUST be a non-empty string
 * Examples:
   * s3
+  * "google.firestore.document"
```

Comment:
Not sure which way we want to go (quotes vs no no-quotes) but since the rest of the spec (as of now) doesn't have quotes on these examples can you remove them?  If you think we should have quotes on all then let's open up a separate issue/PR and get them all in one shot.